src/gui/addnewtorrentdialog.cpp

```diff
@@ -205,6 +206,12 @@ void AddNewTorrentDialog::show(QString source, QWidget *parent)
     }
 }
 
+void AddNewTorrentDialog::show(QString source, QWidget *parent)
+{
+    BitTorrent::AddTorrentParams defaultParams;
```

Comment:
I like being explicit in the intent of a line of code, and using the named variable makes it more clear what I'm doing in this version of `AddNewTorrentDialog::show`. However I can see how it's being more explicit than is strictly necessary so I will remove the variable.